tests/Fixer/Import/SingleImportPerStatementFixerTest.php

```diff
@@ -272,6 +272,22 @@ public function provide70Cases()
         ];
     }
 
+    public function atestMessyComments()
```

Comment:
hint: I do the same, but with a TODO prefix. it's easier to spot accidentally left one